Calculator.js

```diff
@@ -1,18 +1,19 @@
-
 function sum(a, b) {
-    return a + b;
-};
+  return a + b;
+}
 function sub(a, b) {
-    return a - b;
-};
+  return a - b;
+}
 function mul(a, b) {
```

Comment:
Try do it on a Single Function 
```suggestion
function mul(a, b) {
```